packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md

```diff
@@ -0,0 +1,709 @@
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT"
+                ]
+            }
+        },
+        "methods": {
+            "href": "https://{yourOktaDomain}/api/v1/authenticators/aut1nd8PQhGcQtSxB0g4/methods",
+            "hints": {
+                "allow": [
+                    "GET"
+                ]
+            }
+        },
+        "deactivate": {
+            "href": "https://{yourOktaDomain}/api/v1/authenticators/aut1nd8PQhGcQtSxB0g4/lifecycle/deactivate",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        }
+    }
+}
+```
+
+##### Error response example %optional%
+
+%Here you can show any errors that are specific to this API. One error example per section%
+
+```http
+
+```
+
+
+
+### Activate an Authenticator
+
+<ApiOperation method="post" url="/api/v1/authenticators/${authenticatorId}/lifecycle/activate" />
+
+Activates an Authenticator
+
+
+#### Request path parameters
+
+| Parameter          | Type   | Description                                            |
+| ------------------ | ------ | ------------------------------------------------------ |
+| `authenticatorId`  | String | The Authenticator's unique identifier.                 |
+
+#### Request body
+
+N/A
+
+#### Response body
+
+Returns a [Authenticator](#authenticator-object).
+
+#### Usage example
+
+This request would set the `status` of the specified Authenticator to `ACTIVE`
+
+##### Request
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://{yourOktaDomain}/api/v1/authenticators/aut1nd8PQhGcQtSxB0g4/lifecycle/activate"
+```
+
+##### Response
+
+```json
+{
+    "type": "security_key",
+    "id": "aut1nd8PQhGcQtSxB0g4",
+    "key": "webauthn",
+    "status": "ACTIVE",
+    "name": "Security Key or Biometric",
+    "created": "2020-07-26T21:16:37.000Z",
+    "lastUpdated": "2020-07-26T21:59:33.000Z",
+    "_links": {
+        "self": {
+            "href": "https://{yourOktaDomain}/api/v1/authenticators/aut1nd8PQhGcQtSxB0g4",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT"
+                ]
+            }
+        },
+        "methods": {
+            "href": "https://{yourOktaDomain}/api/v1/authenticators/aut1nd8PQhGcQtSxB0g4/methods",
+            "hints": {
+                "allow": [
+                    "GET"
+                ]
+            }
+        },
+        "deactivate": {
+            "href": "https://{yourOktaDomain}/api/v1/authenticators/aut1nd8PQhGcQtSxB0g4/lifecycle/deactivate",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        }
+    }
+}
+```
+
+### Deactivate Authenticator
+
+<ApiOperation method="delete" url="/api/v1/" />
```

Comment:
This should be

`<ApiOperation method="post" url="/api/v1/authenticators/${authenticatorId}/lifecycle/deactivate" />`